crypto-ffi/src/CoreCrypto.udl

```diff
@@ -190,6 +190,12 @@ interface CoreCrypto {
     [Throws=CryptoError]
     void clear_pending_group_from_external_commit(ConversationId conversation_id);
 
+    [Throws=CryptoError]
+    sequence<u8> export_secret_key(ConversationId conversation_id, u32 key_length);
+
+    [Throws=CryptoError]
+    sequence<ClientId> export_clients(ConversationId conversation_id);
```

Comment:
I find the method's name misleading. From a consumer POV, they are just reading the list of clients ids within a group. I would prefer `get_client_ids` (maybe @OtaK will have other thoughts)